product_state/__manifest__.py

```diff
@@ -17,6 +17,7 @@
         "views/product_template.xml",
     ],
     "application": False,
+    "installable": False,
```

Comment:
:thinking:

Reply:
This belong to TMP commit too, would be removed when depend PR merged

Reply:
but this module is coming from the depending PR.... what's the point of making it not installable here?

Reply:
If product_state in TMP commit `installable : True`, it will conflict between 2 version make travis fail

Reply:
When we let  `installable = True`, Travis check raise this ERROR: Cannot install odoo14-addon-product-state 14.0.1.0.1 and 14.0.1.0.2, because these package versions have conflicting dependencies.the current version of the module in the repo when depending on a pending PR impacting to it, so usually we set installable = False to avoid fail Travis test. Please advise other way to resolve the issue.

Reply:
This is the error when we don't have `installable = False`, we took the habit of disabling the current version of the module in the repo when depending on a pending PR impacting it, because of the conflict that happens at Travis level, see here for example [here](https://app.travis-ci.com/github/OCA/product-attribute/jobs/547095331)
Please advise other way to resolve the issue.